src/components/NotFound/NotFound.tsx

```diff
@@ -1,53 +1,44 @@
-import ListItem from '@material-ui/core/ListItem';
+import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
-import withWidth, { isWidthUp, WithWidthProps } from '@material-ui/core/withWidth';
+import styled from 'react-emotion';
 import React, { useCallback } from 'react';
-import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
+
+import Button from '../../muiComponents/Button';
 
 import PackageImg from './img/package.svg';
-import { Card, EmptyPackage, Heading, Inner, List, Wrapper } from './styles';
 
-export const NOT_FOUND_TEXT = `Sorry, we couldn't find it...`;
-export const LABEL_NOT_FOUND = `The page you're looking for doesn't exist.`;
-export const LABEL_FOOTER_NOT_FOUND = 'Perhaps these links will help find what you are looking for:';
+export const NOT_FOUND_TEXT = "Sorry, we couldn't find it...";
+export const LABEL_NOT_FOUND = "The page you're looking for doesn't exist.";
 
-export type NotFoundProps = RouteComponentProps & WithWidthProps;
+const EmptyPackage = styled('img')({
+  width: '150px',
+  margin: '0 auto',
+});
 
-const HOME_LABEL = 'Home';
+const StyledHeading = styled(Typography)({
+  color: '#4b5e40',
+  marginBottom: 16,
+});
 
-const renderSubTitle = (): JSX.Element => (
-  <Typography variant="subtitle1">
-    <div>{LABEL_NOT_FOUND}</div>
-    <div>{LABEL_FOOTER_NOT_FOUND}</div>
-  </Typography>
-);
+const NotFound: React.FC = () => {
+  const history = useHistory();
 
-const NotFound: React.FC<NotFoundProps> = ({ history, width }) => {
   const handleGomHome = useCallback(() => {
     history.push('/');
   }, [history]);
 
-  const renderList = (): JSX.Element => (
-    <List>
-      <ListItem button={true} divider={true} onClick={handleGomHome}>
-        {HOME_LABEL}
-      </ListItem>
-    </List>
-  );
-
-  /* eslint-disable @typescript-eslint/no-non-null-assertion */
   return (
-    <Wrapper data-testid="404">
-      <Inner>
-        <EmptyPackage alt="404 - Page not found" src={PackageImg} />
-        <Heading className="not-found-text" variant={isWidthUp('sm', width!) ? 'h2' : 'h4'}>
-          {NOT_FOUND_TEXT}
-        </Heading>
-        {renderSubTitle()}
-        <Card>{renderList()}</Card>
-      </Inner>
-    </Wrapper>
+    <Box alignItems="center" data-testid="404" display="flex" flexDirection="column" flexGrow={1} justifyContent="center" p={2}>
+      <EmptyPackage alt="404 - Page not found" src={PackageImg} />
+      <StyledHeading className="not-found-text" variant="h4">
+        {NOT_FOUND_TEXT}
+      </StyledHeading>
+      <Button onClick={handleGomHome} variant="contained">
+        {'Go to the home page'}
```

Comment:
Extract them outside as constants in the same file, so we can use them on testing with react testing library.

Reply:
OK...till we implement i18n 😉